cppcache/src/PdxType.hpp

```diff
@@ -205,10 +205,25 @@ class PdxType : public internal::DataSerializableInternal,
   // This is for PdxType as key in std map.
   bool operator<(const PdxType& other) const;
```

Comment:
If we aren't going to use the std::map anymore perhaps we should remove this `operator<`.

Reply:
done